### ISSUE TITLE
Video Scrubbing not working in chrome

### ISSUE THREAD

**Issue by Jeffrey-FB** [BookStackApp/BookStack#5088]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

I'm trying to scrub in Bookstack
Works flawlessly in Firefox but not in anything else.
Will not scrub at all, just keeps flipping back to the beginning of the page.

My setting are a bit high.
This is all an EC2 instance in AWS with local nvme attached storage.
This is in a private cloud. i.e. ping is 1-2ms

```
nano /etc/php/8.3/fpm/php.ini
#A well as here
/etc/php/8.3/cli/php.ini
#Edit these entries
post_max_size = 1024M
upload_max_filesize = 1024M
memory_limit = 512M
```

```
/var/www/bookstack/.env
# Added
# File Upload Limit
# Maximum file size, in megabytes, that can be uploaded to the system.
FILE_UPLOAD_SIZE_LIMIT=250
```

Why is Chrome not scrubbing, is there a setting i need to enable or disable?

### Exact BookStack Version

v24.05.2

### Hosting Environment

Ubuntu 24.04 LTS
php 8.3

**Comment by ssddanbrown** [comment]
Hi @Jeffrey-FB,
- How was the video uploaded to BookStack (or is it located elsewhere)?
- How was the video added to the page?

**Comment by Jeffrey-FB** [comment]
I as this [https://user-images.githubusercontent.com/1685517/41506056-e43bc3c8-7233-11e8-83a3-348ae43345a8.gif](url)
I've enabled 3rd party cookie's and this has helped a little, but only for some users.

So local upload.

**Comment by ssddanbrown** [comment]
@Jeffrey-FB Can you instead try just directly dragging the attachment card into the page for the video?
As shown here: https://www.bookstackapp.com/images/2024/02/bookstack-video-attachment-drag-drop.mp4

**Comment by Jeffrey-FB** [comment]
Hey @ssddanbrown 

It doesn't make a difference, the enabled cookies now doesn't seem to do anything anymore.
It doesn't seem to do the caching or prefetch like it does in firefox. 

Not sure if this is suppose to be working or at least not giving errors?
I cannot seem to resolve it if it should be working.
![Screenshot 2024-07-03 104420](https://github.com/BookStackApp/BookStack/assets/89766127/9dc12799-ce95-4deb-b804-966cd70628a1)

**Comment by ssddanbrown** [comment]
> Not sure if this is suppose to be working or at least not giving errors?

I'm assuming that's in relation to the screenshot? If so, the back/forward cache isn't really relevant here.

- What size of video does this occur with?
- Are you using s3 storage with BookStack at all?
- When viewing the page, if you right-click the video and copy video URL into a new tab (or select to open video in new tab) does it work there?
  - Can you share the video URL (can hide/remove the domain if needed, just need to double check the path)?

Just double checked on my dev instance via chromium, and it seems to be working, so might come down to specifics.

Could be helpful to see a screenshot of the "network" tab of the devtools, when starting the video and scrubbing, after the tab was open, with the latest (bottom) requests in view.

**Comment by Jeffrey-FB** [comment]
Hey @ssddanbrown 

I've limited video's to 250MB
But the size doesn't matter, I've uploaded files between 13MB to 240MB and they behave the same.
Nor does the length of the video.

Path for the 240MB vid
'https://bookstack.my.domain.com/attachments/7'

No not using S3 for anything.
Locally nvme drive on the host.

Again all the scrubbing works fine in firefox

Screen shot of the network tab in dev tools
![Screenshot 2024-07-04 095808](https://github.com/BookStackApp/BookStack/assets/89766127/1e896b01-c870-4fd4-a886-3060d3a44f1f)

**Comment by Jeffrey-FB** [comment]
Hey @ssddanbrown 

So it turns out it's the default player within Chrome.
We test out with another extension, which scrubs okay.
Video play up every now and then.

Not sure if there is anything you can do on your end to resolve this?

But just so you know.
Common issue across all Chromium based platforms, Chrome, edge, opera, brave.... they all behaving the same.

**Comment by ssddanbrown** [comment]
> Not sure if there is anything you can do on your end to resolve this?

Yeah, not sure.
I just double checked via edge 126 (Still on Linux) and seems to work fine.
Could be due to some edge case (specific video encoding, OS, specific chrome base version, network conditions etc...).
I see some streaming attempts in the network tab screenshot, but hard to know exact scenario without digging deeper into specifics.
Could maybe also fundamentally be some issues with how I built the streaming, but I'd need to be able to reproduce really which I havn't been able to do so far.

**Comment by Jeffrey-FB** [comment]
@ssddanbrown 

Appreciate all the help with this.
I'd be great if I could at least give you some extra info if you prepared to investigate further?

Let me me know what you need from me and I'll provide as much as I can.

**Comment by ssddanbrown** [comment]
Hi @Jeffrey-FB,
Sorry for my lack of response, this dropped off my radar.

I had another user report this, and they were able to help pinpoint the details where this occurred to help find the cause.
So this should be addressed as part of the changes in #5342, which is due to be part of the next patch release (v24.10.2 due in the next few days, or maybe today) but if things are the same, after updating to that version, just comment and this can be re-opened.
